File: workspace/pset6/mario/less/mario.py
import sys
from cs50 import get_int

if len(sys.argv) != 1:
    sys.exit("Usage: python mario1.py")

height = get_int("Height: ")

if height < 1:
    height = get_int("Height: ")

if height > 8:
    height = get_int("Height: ")

for i in range(height):
    spaces = height - i
    for j in range(spaces - 1):
        print(" ", end='')
        j = j + 1
    for k in range(height - spaces + 1):
        print("#", end='')
        k = k + 1
    print("")
    i = i + 1

File: workspace/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    int spaces;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        spaces = n - i - 1;

        for (int j = 0; j < spaces; j++)
        {
            printf(" ");
        }

        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}